backend/isoko-backend/src/handlers/user/get-user-object.js

```diff
@@ -10,12 +12,39 @@ exports.getUserObjectHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const { pk } = event.pathParameters;
+
+   if (pk == null) {
+      throw new Error(
+         `Missing query parameter 'pk'. Request URL format: GET/user/{pk}`
+      );
+   }
 
```

Comment:
here and throughout - instead of `throw new Error`, throw a 400 instead for more helpful error logging: 
`return {
        statusCode: 400,
        body: {
           error: `Missing query parameter 'pk'. Request URL format: GET/user/{pk}`,
        },
     };`